ci/cpu/build.sh

```diff
@@ -15,6 +15,11 @@ export HOME=$WORKSPACE
 # Switch to project root; also root of repo checkout
 cd $WORKSPACE
 
+# If nightly build, append current YYMMDD to version
+if [[ "$BUILD_MODE" = "branch" && "$SOURCE_BRANCH" = branch-* ]] ; then
+  export VERSION_SUFFIX=`date +%y%m%d`
+fi
+
 # Get latest tag and number of commits since tag
 export GIT_DESCRIBE_TAG=`git describe --abbrev=0 --tags`
 export GIT_DESCRIBE_NUMBER=`git rev-list ${GIT_DESCRIBE_TAG}..HEAD --count`
```

Comment:
Unrelated question, do we need to explicitly export these? I was under the impression that these variables are already populated for conda builds of git repos.

Reply:
Yeah both of these are set by Conda. Some repos still seem to [set these]( https://github.com/rapidsai/rapids-cmake/blob/9a1beebf9966145615f3b13f9bf3452ba580d5c4/ci/gpu/build.sh#L26-L27 ). Perhaps we should raise a new thread to discuss this)